### PR TITLE
V3.7.2

### PR DESCRIPTION
**Pull request by ocefpaf**
Trying to port @mingwandroid's changes to our recipe. At the moment this failed on AppVeyor with:

```
Build started 3/14/2019 1:52:33 PM.
     1>Project "%SRC_DIR%\PCbuild\pcbuild.proj" on node 1 (Build target(s)).
     1>Project "%SRC_DIR%\PCbuild\pcbuild.proj" (1) is building "%SRC_DIR%\PCbuild\pythoncore.vcxproj" (2) on node 1 (Build target(s)).
     2>%SRC_DIR%\PCbuild\python.props(85,31): error MSB4184: The expression "[System.Version]::Parse('')" cannot be evaluated. Version string portion was too short or too long. [%SRC_DIR%\PCbuild\pythoncore.vcxproj]
     2>Done Building Project "%SRC_DIR%\PCbuild\pythoncore.vcxproj" (Build target(s)) -- FAILED.
     1>Done Building Project "%SRC_DIR%\PCbuild\pcbuild.proj" (Build target(s)) -- FAILED.
Build FAILED.
       "%SRC_DIR%\PCbuild\pcbuild.proj" (Build target) (1) ->
       "%SRC_DIR%\PCbuild\pythoncore.vcxproj" (Build target) (2) ->
         %SRC_DIR%\PCbuild\python.props(85,31): error MSB4184: The expression "[System.Version]::Parse('')" cannot be evaluated. Version string portion was too short or too long. [%SRC_DIR%\PCbuild\pythoncore.vcxproj]
```

Pinging @isuruf, @jjhelmus, @msarahan, and @mariusvniekerk who may see what I missed.

### CODE REVIEW

**Comment by ocefpaf** [comment]
@isuruf do you have any advice for us here on Windows?

**Comment by isuruf** [comment]
@ocefpaf, can you add @mingwandroid to maintainers in master? I can do it if you are ok

**Comment by ocefpaf** [comment]
> @ocefpaf, can you add @mingwandroid to maintainers in master? I can do it if you are ok

Sure. I can do that. BTW, thanks for the assistance!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -67,72 +65,83 @@ source:
...
+    on both a small and large scale.
+  doc_url: https://www.python.org/doc/versions/
+  doc_source_url: https://github.com/python/pythondotorg/blob/master/docs/source/index.rst
+  dev_url: https://docs.python.org/devguide/
 
 extra:
   recipe-maintainers:
     - jakirkham
     - jjhelmus
+    - mingwandroid
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Hey @isuruf looks like @mingwandroid is already here :smile:

Comment (isuruf) [reply recipe/meta.yaml]: Yeah, but this is not merged yet. I added him on master.

Comment (ocefpaf) [reply recipe/meta.yaml]: Ah OK. I was planning on merging this as soon as OS X passes.

**Pull request merged**